cmd/apiserver-boot/boot/create/resource.go

```diff
@@ -502,11 +503,11 @@ func (p *{{ lower .Kind }}Plugin) ValidateInitialization() error {
 	return nil
 }
 
-func (p *{{ lower .Kind }}Plugin) Admit(ctx context.Context, a admission.Attributes, o admission.ObjectInterface) error {
+func (p *{{ lower .Kind }}Plugin) Admit(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error {
 	return nil
 }
 
-func (p *{{ lower .Kind }}Plugin) Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterface) error {
+func (p *{{ lower .Kind }}Plugin) Validate(ctx context.Context, a admission.Attributes, o admission.ObjectInterfaces) error {
```

Comment:
dang, oversight. LGTM now, but can you drop the non-ff merge commit from the pull?